org.sonarlint.eclipse.ui/intro/RELEASE_NOTES.html

```diff
@@ -20,7 +20,21 @@ <h1>{VERSION}</h1>
 -->
 
 <div>
-	<h1>10.4</h1>
+	<h1>10.5</h1>
+	<p>
+		See the <a href="https://github.com/SonarSource/sonarlint-eclipse/releases">GitHub Release page</a>
+		<ul>
+			<li>Latest release notes and awareness about new update of SonarLint directly inside the IDE.</li>
+			<li>Dataflow bug detection support for Java and Python in Connected Mode.</li>
```

Comment:
Not sure which direction you want to take with this, but in other IDE we usually stay 'user friendly' when writing change notes. Meaning that we don't talk about 'Dataflow bug detection support for Java and Python' for example, but rather 'More complex bug detection for Java and Python', because DBD is a very Sonar specific term

Reply:
Mhm, Idk. I copied that from the VSCode changelog [HERE](https://github.com/SonarSource/sonarlint-vscode/blob/master/CHANGELOG.md#420).